aws-rds-dbcluster/src/test/java/software/amazon/rds/dbcluster/CreateHandlerTest.java

```diff
@@ -481,4 +483,39 @@ public void handleRequest_CreateDbCluster_DomainNotFoundException() {
 
         verify(rdsProxy.client(), times(1)).createDBCluster(any(CreateDbClusterRequest.class));
     }
+
+    @Test
+    public void handleRequest_CreateDbCluster_StorageTypeNotSupportedException() {
```

Comment:
nit: Is it possible to replicate Parametric tests from DBInstance in DBCluster resource?